pyproject.toml

```diff
@@ -13,6 +13,8 @@ gunicorn = "^20.1.0"
 django-phonenumber-field = {extras = ["phonenumberslite"], version = "^7.0.2"}
 django-extensions = "^3.2.1"
 psycopg2-binary = "^2.9.5"
+djangorestframework = "^3.14.0"
+drf-yasg = "^1.21.5"
```

Comment:
From drf_yasg docs:

> If you are looking to add Swagger/OpenAPI support to a new project you might want to take a look at [drf-spectacular](https://github.com/tfranzel/drf-spectacular), which is an actively maintained new library that shares most of the goals of this project, while working with OpenAPI 3.0 schemas.

Should we maybe use drf_spectacular then?

Reply:
Yeah, fixed